source/docs/software/support/support-resources.rst

```diff
@@ -29,6 +29,16 @@ CTRE Support
 
 Support for Cross The Road Electronics components (Pneumatics Control Module, Power Distribution Panel, Talon SRX, and Voltage Regulator Module) is provided via the e-mail address support@crosstheroadelectronics.com
 
+REV Robotics Support
+------------
```

Comment:
Gotta extend this to be the num characters of the title.

Reply:
Oops - missed that.  Trying to do 20 things at once and something broke.  Sadly it was this build.

Reply:
No worries. I'll push the link changes for ya when I get off work ;)

Reply:
Hopefully it's fixed now but if not then let me know.  I'm having to do this from a browser at the moment and I need to get setup for editing this locally.

Reply:
Yep. I'm more thinking of my other comment. We can actually turn these into clickable links that open a users phone or email client with "mailto:" and "tel" links

Reply:
Gotcha.  Yeah, if you can make the REV one pop up with a subject line of "Hi Greg" that would be great. ;)